Plugins/Reqnroll.Verify/Reqnroll.Verify.ReqnrollPlugin/build/Reqnroll.Verify.targets

```diff
@@ -2,8 +2,5 @@
   <PropertyGroup>
     <_VerifyGeneratorPluginFramework Condition=" '$(MSBuildRuntimeType)' == 'Core'">netstandard2.0</_VerifyGeneratorPluginFramework>
```

Comment:
This two property setting is not needed (used for generators), but instead you need:

```
    <_VerifyRuntimePlugin Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">net472</_VerifyRuntimePlugin>
    <_VerifyRuntimePlugin Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' ">net6.0</_VerifyRuntimePlugin>
    <_VerifyRuntimePluginPath>$(MSBuildThisFileDirectory)..\lib\$(_VerifyRuntimePlugin)\Reqnroll.Verify.ReqnrollPlugin.dll</_VerifyRuntimePluginPath>
```

This is basically selecting the runtime plugin based on `$(TargetFrameworkIdentifier)` (the compilation target). The `$(MSBuildRuntimeType)` is the framework used for compilation, that is needed to decide which generator plugin version we need to load.

We need all this for copying the runtime plugin dll to the output folder (see props file), because Reqnroll will search for it there and in modern .NET the reference assemblies are not copied to the output folder by default.

Reply:
right :)